DoLah/Models/Calendar.h

```diff
@@ -20,7 +22,10 @@ namespace DoLah {
         void setDoneTask(int, bool=true);
 		void updateTask(int, AbstractTask*);
 		void clearTasks();
-		void search(std::string);
+        void search(std::string);
+        void sortTasks(std::vector<AbstractTask*>&);
+        static bool taskCompare(AbstractTask*, AbstractTask*);
+        static std::vector<std::tm> getDates(AbstractTask*);
```

Comment:
ok then please elaborate further. What you are disagree and a potential alternative and etc etc.